docs/artifacts/why-639-cannot-merge.html

```
@@ -0,0 +1,110 @@
+<title>PR 639 Merge Block</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)